City-of-Helsinki/drupal-infofinland
# IFU-951 missing current revision

# [IFU-951](https://helsinkisolutionoffice.atlassian.net/browse/IFU-951)
Revision overview form is missing current revision. 

## What was done
Custom patch to force the first revision to be visible on revision overview form.

## How to reproduce bug before you apply the fix:
Create new content, add a field, save
Create a translation for the content and save
Add a new paragraph to the translation and save as new revision
Go check the original language revision overview form. You should be missing "current revision" row from the table

## How to test
Follow the reproduction steps. Now all translations should have the current revision visible always.
Try rollbacking revisions and saving and rollbacking again and see that it always shows correct revision as first (for all languages)


[IFU-951]: https://helsinkisolutionoffice.atlassian.net/browse/IFU-951?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
